packages/dcos-integration-test/extra/test_dcos_diagnostics.py

```diff
@@ -212,38 +212,38 @@ def test_systemd_units_health(dcos_api_session):
         raise AssertionError('\n'.join(unhealthy_output))
 
 
-def test_dcos_diagnostics_units_unit(dcos_api_session):
+def test_dcos_diagnostics_units_unit(dcos_api_session: DcosApiSession) -> None:
     """
     test a unit response in a right format, endpoint: /system/health/v1/units/<unit>
     """
     for master in dcos_api_session.masters:
         units_response = check_json(dcos_api_session.health.get('/units', node=master))
-        pulled_units = list(map(lambda unit: unit['id'], units_response['units']))
+        pulled_units = list(map(lambda unit: unit['id'], units_response['units']))  # type: ignore
```

Comment:
Isn't `pulled_units` a list?

Reply:
it is but this ignore cover other issue:
```
test_dcos_diagnostics.py:221: error: Returning Any from function declared to return "object"
```